.pre-commit-config.yaml

```diff
@@ -0,0 +1,34 @@
+repos:
```

Comment:
i would recommend finding a pre-commit hook for docker at https://pre-commit.com/hooks.html  and adding that to this file.  something like `dockerfilelint` would be good.